docs/react-testing-library.md

```diff
@@ -0,0 +1,72 @@
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [await-async-events](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/await-async-events.md)<br />Enforce promises from async event methods are handled                                 | <details><summary>eventModule: userEvent</summary><pre lang="json"><code>{&#13;  "eventModule": "userEvent"&#13;}</code></pre></details> |   🔧    |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [await-async-queries](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/await-async-queries.md)<br />Enforce promises from async queries to be handled                                   |                                                                                                                                          |         |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [await-async-utils](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/await-async-utils.md)<br />Enforce promises from async utils to be awaited properly                                |                                                                                                                                          |         |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [no-dom-import](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/no-dom-import.md)<br />Disallow importing from DOM Testing Library                                                     | <details><summary>react</summary><pre lang="json"><code>"react"</code></pre></details>                                                   |   🔧    |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [no-global-regexp-flag-in-query](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/no-global-regexp-flag-in-query.md)<br />Disallow the use of the global RegExp flag (/g) in queries    |                                                                                                                                          |   🔧    |           |
+| [![testing-library](./icons/other/testing-library.png)](undefined) | [no-manual-cleanup](https://github.com/testing-library/eslint-plugin-testing-library/tree/main/docs/rules/no-manual-cleanup.md)<br />Disallow the use of `cleanup`                                                           |                                                                                                                                          |         |           |
```

Comment:
Sounds like the extra cleanup is merely redundant, so this could be a warning.

Reply:
Sure, changed it.